src/ctapipe_io_lst/calibration.py

```diff
@@ -34,6 +36,11 @@
     'LSTR0Corrections',
 ]
 
+def to_native(data):
+    if not data.dtype.isnative:
+        data = data.byteswap()
+        data = data.view(data.dtype.newbyteorder("="))
+        return data
```

Comment:
```suggestion
    return data
```
Otherwise, the function will return `None` in the case of the data already being native. 
You should add a test for this function.